### PR TITLE
added cve-2022-21907 audits

### PR DESCRIPTION
**Pull request by jturpin001**

### CODE REVIEW

**Review by rjcomp — APPROVED** [review]

File: cve-2022-21907/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# CVE-2021-40444
```
Comment (marbe9312) [inline cve-2022-21907/README.md]: wrong cve in README

File: cve-2022-21907/cve-2022-21907.audit [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          value_type  : POLICY_DWORD
+          value_data  : 0
+          reg_key     : "HKLM\System\Currentcontrolset\Services\Http\Parameters"
+          reg_item    : "EnableTrailerSupport"
+          reg_option  : CAN_BE_NULL
+        </custom_item>
+      </then>
+
+      <else>
+        <report type:"PASSED">
```
Comment (marbe9312) [inline cve-2022-21907/cve-2022-21907.audit]: may want to change this to a WARNING so people don't see it and think they are not vulnerable.

File: cve-2022-21907/README.md (hunk 1 above)
Comment (jturpin001) [reply cve-2022-21907/README.md]: fixed

File: cve-2022-21907/cve-2022-21907.audit (hunk 2 above)
Comment (jturpin001) [reply cve-2022-21907/cve-2022-21907.audit]: done

**Review by rjcomp — APPROVED** [review]

**Pull request merged**